pkg/proxy/proxy.go

```diff
@@ -61,6 +61,11 @@ func LoadConfigFromEnv() *types.Config {
 }
 
 func NewOAuthProxy(config *types.Config) (*OAuthProxy, error) {
+	if u, err := url.Parse(config.MCPServerURL); err != nil || u.Scheme != "http" && u.Scheme != "https" {
```

Comment:
Probably better to put this logic into `LoadConfigFromEnv` to make it more encapsulated.

Reply:
Done.